Helper/Config.php

```diff
@@ -55,4 +55,19 @@ public function getToken(?int $storeId = null): ?string
             $storeId
         );
     }
+
+    /**
+     * Return if product canonical url configuration is enabled or not
+     *
+     * @param int|null $storeId
+     * @return string|null
+     */
+    public function isUseProductCanonicalUrlEnabled(?int $storeId = null): bool
+    {
+        return $this->scopeConfig->isSetFlag(
+            self::XML_PATH_PRERENDER_USE_PRODUCT_CANONICAL_URL,
```

Comment:
It doesn't look like this `XML_PATH_PRERENDER_USE_PRODUCT_CANONICAL_URL` constant is defined.